sheepdog/transactions/upload/entity.py

```diff
@@ -224,7 +237,14 @@ def _get_node_create(self, skip_node_lookup=False):
 
         node = cls(self.entity_id)
         if is_data_file:
-            node.acl = self.transaction.get_phsids()
+            # check if open_acl is requested and the node type can be set open
+            if self.doc.get('open_acl', None) and self._config.get('IS_GDC', False):
```

Comment:
so we have an open_acl field in the gdcdictionary schema? is it a boolean(then would be `self.doc.get('open_acl') is True`?

Reply:
We do not. It's just a field allowed when submitting.

Reply:
oh, how does it pass the dictionary schema check when this field is given during submission?